charts/delphai-deployment/templates/mapping.yaml

```diff
@@ -46,7 +50,28 @@ spec:
   timeout_ms: 60000
   idle_timeout_ms: 500000
 
-{{ if $.Values.ingress.publicApiEnabled }}
+{{ if $.Values.ingress.publicApiPrefix }}
+
+{{ if or
+  (not (hasPrefix "/" $.Values.ingress.publicApiPrefix))
+  (not (hasSuffix "/" $.Values.ingress.publicApiPrefix))
+}}
+{{- fail "The public API prefix must start and end with a slash" }}
+{{ end }}
+
+{{ $publicApiPrefix := print
+  ( ternary
+    ""
+    ( print "/" $.Release.Name )
+    ( index $.Values.buildMetadata.labels "com.delphai.image.release" )
+  )
+  ( substr
+    0
+    (sub ( len $.Values.ingress.publicApiPrefix ) 1 | int )
+    $.Values.ingress.publicApiPrefix
+  )
```

Comment:
https://helm.sh/docs/chart_template_guide/function_list/#trimprefix

Reply:
Yup, that looks simpler. Rewritten in 7799e64.